pythia/cleaning/tests/test_elo.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import pandas as pd
+from pythia.cleaning import ELO
+from pathlib import Path
+from pythia.seo import Sunspotter
+from sunpy.util import SunpyUserWarning
+
+path = Path(__file__).resolve().parent.parent.parent.parent / "data/elo_test"
```

Comment:
If the file is only used in this test, I probably would save the file in this directory.